backend/python/app/models/comment.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+from . import db
+
+
+class Comment(db.Model):
+
+    __tablename__ = "comments"
+
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    story_translation_id = db.Column(
+        db.Integer, db.ForeignKey("story_translations.id"), index=True, nullable=False
+    )
+
+    user_id = db.Column(
+        db.Integer, db.ForeignKey("users.id"), index=True, nullable=False
```

Comment:
remove `index=True`, database indexes are good for when you'll need to query often based on the column. I dont foresee us querying for comments by user_id

https://en.wikipedia.org/wiki/Database_index#:~:text=A%20database%20index%20is%20a,maintain%20the%20index%20data%20structure.